sirepo/auth_db.py

```diff
@@ -303,13 +301,12 @@ def _migrate_db_file(fn):
 
 
 def _migrate_role_jupyterhub():
```

Comment:
This should get moved out, too. It should be in db_ugprade, and it should be a registered call.